CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+#### [2.0.0](https://github.com/hirosystems/clarinet/compare/v1.8.0...v2.0.0) (2023-10-26)
+
+##### New Features
+
+*  Introduce global ~/.clarinet/clarinetrc.toml (#1208) (bbe26ccf)
```

Comment:
currently it looks for `~/.clarinet/Settings.toml`

Reply:
hey @tippenein 
I mentioned in this follow up PR as well: https://github.com/hirosystems/clarinet/pull/1225

Reply:
Ah! I see you responded here: https://github.com/hirosystems/clarinet/pull/1225

Thanks!